docs/user_stats.rst

```diff
@@ -27,6 +29,50 @@ List of Callbacks
 Function Reference
 ------------------
 
+.. function:: userStats.getStat(name)
+
+
+    :param string name: The 'API Name' of the stat.
+    :returns: (`boolean`)
+        This function returns true upon success if all of the following conditions are met; otherwise, false.
+
+        * :func:`userStats.requestCurrentStats` has completed and successfully returned its callback.
+
+        * The 'API Name' of the specified stat exists in App Admin on the Steamworks website, and the changes are published.
+    :returns: (`boolean?`) If the call is successful, returns a second value, the current value of the stat.
+    :SteamWorks: `GetStat <https://partner.steamgames.com/doc/api/ISteamUserStats#GetStat>`_
+
+    Gets the current value of a users stat.
+
```

Comment:
Add the text about using `getUserStat (missing)` to get other users stats, just like in the `getAchievement` function.